src/apps/content-editor/src/app/components/Editor/Field/Field.js

```diff
@@ -129,17 +129,26 @@ function resolveRelatedOptions(
       );
     })
     .map((itemZUID) => {
-      return {
-        filterValue: items[itemZUID].data[field.name],
-        value: itemZUID,
-        component: (
-          <RelatedOption
-            modelZUID={modelZUID}
-            itemZUID={itemZUID}
-            text={items[itemZUID].data[field.name]}
-          />
-        ),
-      };
+      for (const [key, value] of Object.entries(items[itemZUID].siblings)) {
+        if (items[itemZUID].meta.ZUID === value) {
+          return {
+            filterValue: items[itemZUID].data[field.name],
+            value: itemZUID,
+            component: (
+              <RelatedOption
+                modelZUID={modelZUID}
+                itemZUID={itemZUID}
+                text={
+                  <>
+                    <span>{items[itemZUID].data[field.name]}</span>
+                    <em className={styles.Language}>&nbsp;{key}</em>
+                  </>
+                }
+              />
+            ),
+          };
```

Comment:
multi-lang instance can have N languages and so once we find a match, e.g. are in the `if` block, we can exit the `for .. of` loop early with a `break` statement. This is a minor improvement but a good habit to build.